### ISSUE TITLE
Request: Remove hard-coded margins, padding

### ISSUE THREAD

**Issue by tobydriscoll** [JuliaDocs/Highlights.jl#23]
It looks as though Format.jl sets the CSS border, margins, and padding of a `pre.hljl` selector using hard-coded values. In my testing, a theme cannot override these settings. This complicates downstream usage. Most saliently, the caller cannot set them in the `pre` selector for itself and have those values respected across a variety of containers. (See https://github.com/mpastell/Weave.jl/pull/172#issuecomment-462839397.)

Options would be to remove these values from the `pre.hljl` selector in Format.jl, or make the values changeable via the theme.

**Comment by mortenpi** [comment]
I agree that hard-coded values in the CSS are not ideal. If nothing else, we could make it configurable, so that you can override the values or have a way of opting out of creating the `pre.hljl` block here:

https://github.com/JuliaDocs/Highlights.jl/blob/1f498a64013582b99ff2593c1c6959b1ef5a3fd6/src/Format.jl#L96-L105
